app/src/main/java/com/mapbox/maps/testapp/examples/localization/LocalizationActivity.kt

```diff
@@ -0,0 +1,154 @@
+package com.mapbox.maps.testapp.examples.localization
+
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuItem
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import com.mapbox.maps.MapboxMap
+import com.mapbox.maps.Style
+import com.mapbox.maps.extension.style.localization.localizeLabels
+import com.mapbox.maps.testapp.R
+import kotlinx.android.synthetic.main.activity_map_localization.*
+import kotlinx.android.synthetic.main.activity_map_overlay.mapView
+import java.util.*
+
+/**
+ * Example showcasing usage of localization.
+ */
+class LocalizationActivity : AppCompatActivity() {
+  private lateinit var mapboxMap: MapboxMap
+  private var mapIsLocalized: Boolean = false
+  private var index: Int = 0
+  private val styles =
+    arrayOf(
+      Style.MAPBOX_STREETS,
+      MAPBOX_STREETS_V10
+    )
+
+  private val nextStyle: String
+    get() {
+      return styles[index++ % styles.size]
+    }
+  private lateinit var locale: Locale
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_map_localization)
+    locale = resources.configuration.locale
+    mapIsLocalized = true
+    Toast.makeText(this, R.string.change_language_instruction, Toast.LENGTH_LONG).show()
+    mapboxMap = mapView.getMapboxMap()
+    mapboxMap.loadStyleUri(nextStyle) {
+      it.localizeLabels(locale)
+    }
+    fabStyles.setOnClickListener {
+      val styleUri = nextStyle
+      mapboxMap.loadStyleUri(styleUri) {
+        it.localizeLabels(locale)
+      }
+      Toast.makeText(this, styleUri, Toast.LENGTH_SHORT).show()
+    }
+    fabLocalize.setOnClickListener {
+      mapIsLocalized = if (mapIsLocalized) {
+        mapboxMap.getStyle()?.localizeLabels(Locale.FRENCH)
```

Comment:
shall we toggle between the system language and the language selected in the `OptionsMenu`?